src/serverlessFunction/view.ts

```diff
@@ -169,24 +169,15 @@ export class ServerlessFunctionView implements TreeDataProvider<ExplorerItem>, D
     async getChildren(element?: ExplorerItem): Promise<ExplorerItem[]> {
         let result: ExplorerItem[] = [];
         if (!element) {
-            result = [{
-                kind: 'project',
-                metadata: {
-                    name: this.kubeContext?.namespace,
-                },
-            } as KubernetesObject];
-            this.setCurrentNameSpace(this.kubeContext?.namespace);
-        } else if ('kind' in element) {
-            if (element.kind === 'project') {
-                result = [...await this.serverlessFunction.getLocalFunctions()]
+            result = [...await this.serverlessFunction.getLocalFunctions()]
                 if (result.length === 0) {
                     const functionNode: FunctionObject = {
                         name: 'No Available Functions',
                         context: FunctionStatus.NONE
                     }
                     result = [functionNode]
                 }
-            }
+            this.setCurrentNameSpace(this.kubeContext?.namespace);
```

Comment:
Is this still needed? I think you can get rid of this as well as the other code that watches for namespace changes, such as the file watcher on the `~/.kube/config`, since you no longer need to display the up to date namespace.

Reply:
> Is this still needed? I think you can get rid of this as well as the other code that watches for namespace changes, such as the file watcher on the `~/.kube/config`, since you no longer need to display the up to date namespace.

The latest namespace needed on deploy command. There was a parameter we need to pass

Reply:
As a part of the "remove cycles" PR I made the following change: https://github.com/redhat-developer/vscode-openshift-tools/blob/main/src/serverlessFunction/functions.ts#L242

Reply:
Removed :)